packages/medusa/src/services/order.js

```diff
@@ -418,30 +453,39 @@ class OrderService extends BaseService {
   async createShipment(orderId, fulfillmentId, trackingNumbers, metadata = {}) {
```

Comment:
When does it update the shipment status?

Reply:
I am not sure what you mean... We don't have a shipment status on orders.